EmbeddedScripts.CSharp.Tests/CompiledCodeRunnerTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Xunit;
+using System.Threading.Tasks;
+using EmbeddedScripts.Shared;
+using HelperObjects;
+
+namespace EmbeddedScripts.CSharp.Tests
+{
+    public class CompiledCodeRunnerTests
+    {
+        [Fact]
+        public async Task CorrectCode_ExecutionSucceed()
+        {
+            var code = "var a = 1; var b = 2; var c = a + b;";
+
+            var runner = new CompiledCodeRunner(code, new Container());
+            await runner.RunAsync();
+        }
+
+        [Fact]
+        public async Task RunWithGlobalVariables_Succeed()
+        {
+            var t = new HelperObject();
+
+            var container = new Container();
+            container.Register(t, "t");
```

Comment:
Maybe more fluent way to setup dependencies/runners?
e.g.
`container.Register(a, "a").Register(b, "b")`